lecture_3/03_project/calculator/internal/infrastructure/sqlite/calcBetRepository.go

```diff
@@ -98,7 +122,7 @@ func (r *CalcBetRepository) queryGetBetsByEventID(ctx context.Context, eventId s
 }
 
 // CalcBetWithIDExists returns True if a bet with the provided id exists.
-// If the bet does not exist, an error will not be returned. <- WRONG and I don't know how to fix it.
+// If the bet does not exist, an error will not be returned. <- WRONG and I don't know how to fix it
```

Comment:
I would like to help you, but cant without proper debugging. Method seems fine at the first glance. (Dominik took a look at it as well)

You can ask one of the lecturers on 4th or 5th lecture to help you with this.